gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerUiState.kt

```diff
@@ -81,7 +82,7 @@ internal data class AvatarsSectionUiState(
 
 internal data class AvatarUploadFailure(
     val uri: Uri,
-    val error: String?,
+    val error: ErrorType?,
```

Comment:
I've replaced this because it looks better if the view handles what string should be shown. With the `String` we would need to access `R.string` from the `ViewModel`.